mediaflow_proxy/speedtest/models.py

```
@@ -21,26 +20,20 @@ class UserInfo(BaseModel):
+class MediaFlowServer(BaseModel):
+    url: HttpUrl
+    api_password: Optional[str] = None
+    name: Optional[str] = None
 
 
-class LocationResult(BaseModel):
-    result: Optional[SpeedTestResult] = None
-    error: Optional[str] = None
-    server_name: str
-    server_url: str
+class BrowserSpeedTestConfig(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)